### ISSUE TITLE
TOTP usage

### ISSUE THREAD

**Issue by c4lyp5o** [nocturnalmc/gp2#733]
pentadbir boleh log in menggunakan authenticator app (google authenticator, microsoft authenticator, dll). boleh cuba test.

steps:
1. login ke pentadbir
2. tekan 'Tetapan saya' dekat nama (atas kanan)
3. Aktifkan TOTP
4. Scan qrcode dengan authenticator app
5. Masukkan nombor yg ada dalam input Kod Totp
6. Tekan simpan

Setelah aktif totp, login akan menggunakan kod totp sahaja, email xperlu bukak.

**Comment by nocturnalmc** [comment]
Baik akan di check. Skrg tgh checking date input @c4lyp5o

**Comment by nocturnalmc** [comment]
// TODO
- Toast container show undefined for send to which email when pentadbir login

@c4lyp5o dato aku baru checking halfway. Kalau dato nak fix dlu TODO list ni dato buat lah dlu. Nti aku update commit information kat sini

**Comment by c4lyp5o** [comment]
@nocturnalmc dato, ni ialah kalau pakai totp ke atau kalau pakai email

**Comment by nocturnalmc** [comment]
> @nocturnalmc dato, ni ialah kalau pakai totp ke atau kalau pakai email

Dalam DB totp: false dato

**Comment by c4lyp5o** [comment]
kalau login biasa eh @nocturnalmc

**Comment by nocturnalmc** [comment]
> kalau login biasa eh @nocturnalmc

Yup dato. Dalam email dapat je string otp tu dan boleh login. Cuma display kat toast ja tulis 'undefined'

**Comment by nocturnalmc** [comment]
Yup dato. Dalam email dapat je string otp tu dan boleh login. Cuma display kat toast ja tulis 'undefined' @c4lyp5o

**Comment by c4lyp5o** [comment]
@nocturnalmc baik dato aku di uji coba dan di perbaiki

**Comment by c4lyp5o** [comment]
@nocturnalmc @shukritolahah ni dah fix rsnya. sila test
